lib/Partitioner/Partitioner.cpp

```diff
@@ -1211,7 +1225,6 @@ llvm::Error Partitioner::QuantizationProfilingPartition(
   module_->eraseFunction(F_);
```

Comment:
Is it possible that the function can be renamed and will no longer match the name in partitionConfig?

Reply:
No. If partitionConfig is set, Partitioner will go to user-defined flow without considering other options like "-load-profiling".